### ISSUE TITLE
[Accessibility issue] Vignettes lack alt= attributes

### ISSUE THREAD

**Issue by MichaelChirico** [markmfredrickson/optmatch#227]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in `knitr`; see https://www.rstudio.com/blog/knitr-fig-alt/.

Both of the images flagged were in this vignette:

https://github.com/markmfredrickson/optmatch/blob/d30a3f1ff85f9e0e244b4c4c3f7b3dd95b9a60ac/vignettes/fullmatch-vignette.Rmd

Unfortunately the lines are flagged in the HTML output (lines 519, 621), which won't correspond exactly to the .Rmd file lines, but I assume if you're familiar with the vignette it will be clear which images need it.

**Comment by josherrickson** [comment]
Thank you for flagging this. I'll work on getting this updated soon.

**Comment by benthestatistician** [comment]
Thanks for alterting us to this issue, @MichaelChirico.

As @josherrickson has noted [elsewhere](https://github.com/markmfredrickson/RItools/issues/130), we're seeing errors on CRAN from the same blocks of code that are implicated here.  I'm chiming in to say that if they turn out not to be so simple to fix then I think we should push a maintenance release that temporarily disables these visualizations.  To buy us time to figure out both the errors and the accessibility issue.

**Comment by josherrickson** [comment]
I've added alt-text; next release will include them. Thanks again for the notification.
